res/database/Default/user/kasi.xml

```diff
@@ -1396,7 +1445,7 @@
 			<speed slope="40" max="50" min="30" />
 		</scripttemplate>
 
-		<!--TODO reaktivieren/weglassen?
+		<!--TODO reaktivieren/weglassen? (Warum?)
```

Comment:
no energy left, feel free to activate

Reply:
what?

Reply:
When incorporating Kasi's proposed database entries, I left some commented out.
I guess, all that still needs to be done is deleting the comment.